src/services/document-input-service.js

```diff
@@ -187,32 +187,34 @@ class DocumentInputService {
         sections: finalSections
       };
 
-      lock = await this.lockStore.takeUserLock(room.ownedBy);
+      if (newCdnFiles.length) {
```

Comment:
I also fixed this in the last commit to only check storage quota if there are actually files in the input.